policies/api-consolidation-documentation.md

```diff
@@ -0,0 +1,252 @@
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UpdateUser"
+      responses:
+        "200":
+          description: User successfully updated
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/User"
+        "400":
+          description: Invalid input
+        "404":
+          description: User not found
+        "500":
+          description: Internal Server Error
+
+    delete:
+      summary: Delete a user
+      description: Delete a user by their user ID.
+      parameters:
+        - name: userId
+          in: path
+          required: true
+          description: ID of the user to delete
+          schema:
+            type: string
+      responses:
+        "204":
+          description: User successfully deleted
+        "404":
+          description: User not found
+        "500":
+          description: Internal Server Error
+
+components:
+  schemas:
+    User:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the user
+        name:
+          type: string
+          description: Full name of the user
+        email:
+          type: string
+          description: Email address of the user
+        createdAt:
+          type: string
+          format: date-time
+          description: Timestamp of when the user was created
+
+    NewUser:
+      type: object
+      required:
+        - name
+        - email
+      properties:
+        name:
+          type: string
+          description: Full name of the user
+        email:
+          type: string
+          description: Email address of the user
+
+    UpdateUser:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Full name of the user
+        email:
+          type: string
+          description: Email address of the user
+```
+
+### 2. Documentation Maintenance
+
+- API documentation must be reviewed and updated regularly or when a new version of the API is released.
+- Old or deprecated APIs should be clearly marked in the documentation, with migration guidelines provided where necessary.
+
+### 3. Review Process
+
+- Before merging any code that changes the API, a review of the documentation must be conducted by a peer or senior team member.
+- Pull requests that involve API changes will not be merged unless corresponding updates to the API documentation are present.
+
+### 4. Versioning
+
+- API documentation must follow the same versioning system as the API. Any major, minor, or patch release should have its own corresponding documentation.
+- All versions of the API documentation should be archived and accessible.
+
+### 5. Hosting and Access
+
+- API documentation must be accessible to authorized users and hosted in a secure environment (e.g., **booky-docs**) bucket.
```

Comment:
Weird to PR here but let's change bucket name pleeease.

Reply:
resolved https://github.com/scrambledeggs/cookbook/pull/12/commits/88dce3ba48e160890ff993398222e4a82702568f